tests/misra/test_misra.sh

```diff
@@ -63,3 +74,12 @@ cppcheck $PANDA_OPTS -DSTM32H7 -DSTM32H725xx $PANDA_DIR/board/main.c
 
 printf "\n${GREEN}Success!${NC} took $SECONDS seconds\n"
 
+
+# ensure list of checkers is up to date
+cd $DIR
+if ! cmp -s checkers_report.txt checkers.txt; then
```

Comment:
I think it would be nicer to just overwrite the file, then use `git` to check for a diff. Then, it's just `git add` to update it.

Reply:
I agree, I was thinking that might be better.

Reply:
Another issue is that I accidentally pass an absolute path `$PANDA_DIR` to the report.  
I think I could skip $PANDA_DIR and just call `cppcheck $PANDA_OPTS -DSTM32F4 -DSTM32F413xx /oard/main.c`, but that somehow enables bunch of new violations.

Reply:
oh - let's get those fixed :)